worlds/sc2wol/Options.py

```diff
@@ -175,6 +175,15 @@ class ExtendedItems(Toggle):
     display_name = "Extended Items"
 
 
+class MaxNbUpgrades(Range):
+    """Set a maximum to the number of upgrades a unit/structure can have."""
+    display_name = "Maximum number of upgrades per unit/structure"
+    range_start = -1
```

Comment:
Is -1 unlimited? If so, document it
I'd also document that most units have usually between 4 and 6 upgrades to give an idea to the user about correct value to set

Reply:
Ohhhh! Good catch!

Reply:
Applied!